### ISSUE TITLE
Broken Link for instruction on how to link using own mapView

### ISSUE THREAD

**Issue by maronrubio** [Appolica/InteractiveInfoWindowAndroid#23]
Hi Could you please fix the link on how to link the mapview to your interactiveinfowindow without using your fragment. It is giving me issues on implementing Google Marker Cluster

**Comment by maronrubio** [comment]
Were you guys able to implement the Google Marker Cluster with the Appolica map fragment?

I tried to put the MapInfoWindowFragment on initialization of GoogleMarkerCluster but still I can't get it to work. At first I thought it was because I was referencing a different activity which doesn't have the map but still no luck.

I also tried creating a new MapInfoWindowFragment which i would then use for the initialization of the Google Marker Cluster but no luck.

`mClusterManager = new ClusterManager<MarkerClusterItem>(mapFragment.getActivity(), gMap);`

 Was hoping if you guys have any solution or idea on how I can achieve what I want thanks a lot in advance.
